src/python/k4a/src/k4a/_bindings/k4a.py

```diff
@@ -0,0 +1,525 @@
+k4a_capture_set_depth_image.restype = None
+k4a_capture_set_depth_image.argtypes=(_CaptureHandle, _ImageHandle)
+
+
+#K4A_EXPORT void k4a_capture_set_ir_image(k4a_capture_t capture_handle, k4a_image_t image_handle);
+k4a_capture_set_ir_image = _k4a_lib.k4a_capture_set_ir_image
+k4a_capture_set_ir_image.restype = None
+k4a_capture_set_ir_image.argtypes=(_CaptureHandle, _ImageHandle)
+
+
+#K4A_EXPORT void k4a_capture_set_temperature_c(k4a_capture_t capture_handle, float temperature_c);
+k4a_capture_set_temperature_c = _k4a_lib.k4a_capture_set_temperature_c
+k4a_capture_set_temperature_c.restype = None
+k4a_capture_set_temperature_c.argtypes=(_CaptureHandle, _ctypes.c_float)
+
+
+#K4A_EXPORT float k4a_capture_get_temperature_c(k4a_capture_t capture_handle);
+k4a_capture_get_temperature_c = _k4a_lib.k4a_capture_get_temperature_c
+k4a_capture_get_temperature_c.restype = _ctypes.c_float
+k4a_capture_get_temperature_c.argtypes=(_CaptureHandle,)
+
+
+#K4A_EXPORT k4a_status_t k4a_image_create(k4a_image_format_t format,
+#                                         int width_pixels,
+#                                         int height_pixels,
+#                                         int stride_bytes,
+#                                         k4a_image_t *image_handle);
+k4a_image_create = _k4a_lib.k4a_image_create
+k4a_image_create.restype = EStatus
+k4a_image_create.argtypes=(_ctypes.c_int, _ctypes.c_int, _ctypes.c_int, _ctypes.c_int, _ctypes.POINTER(_ImageHandle))
+
+
+
+#K4A_EXPORT k4a_status_t k4a_image_create_from_buffer(k4a_image_format_t format,
+#                                                     int width_pixels,
+#                                                     int height_pixels,
+#                                                     int stride_bytes,
+#                                                     uint8_t *buffer,
+#                                                     size_t buffer_size,
+#                                                     k4a_memory_destroy_cb_t *buffer_release_cb,
+#                                                     void *buffer_release_cb_context,
+#                                                     k4a_image_t *image_handle);
+k4a_image_create_from_buffer = _k4a_lib.k4a_image_create_from_buffer
+k4a_image_create_from_buffer.restype = EStatus
+k4a_image_create_from_buffer.argtypes=(
+    _ctypes.c_int, _ctypes.c_int, _ctypes.c_int, _ctypes.c_int, _ctypes.POINTER(_ctypes.c_uint8),
+    _ctypes.c_ulonglong, _memory_allocate_cb, _ctypes.c_void_p, _ctypes.POINTER(_ImageHandle))
+
+
+
+#K4A_EXPORT uint8_t *k4a_image_get_buffer(k4a_image_t image_handle);
+k4a_image_get_buffer = _k4a_lib.k4a_image_get_buffer
+k4a_image_get_buffer.restype = _ctypes.POINTER(_ctypes.c_uint8)
+k4a_image_get_buffer.argtypes=(_ImageHandle,)
+
+
+#K4A_EXPORT size_t k4a_image_get_size(k4a_image_t image_handle);
+k4a_image_get_size = _k4a_lib.k4a_image_get_size
+k4a_image_get_size.restype = _ctypes.c_ulonglong
```

Comment:
there is a ctpyes c_size_t
That might be more extensible if we want to support both 32 and 64 bit